www-apps/radicale/radicale-1.1.6.ebuild

```diff
@@ -0,0 +1,72 @@
+# Copyright 1999-2015 Gentoo Foundation
```

Comment:
please fix the header.
Did you run repoman full?

Reply:
done, i think i did and it did not complain